its/ruling/src/test/resources/expected/python-S6395.json

```diff
@@ -0,0 +1,43 @@
+{
+'project:biopython/Bio/SearchIO/ExonerateIO/exonerate_text.py':[
+32,
+32,
+36,
+36,
```

Comment:
To be honest, I'm not sure about those issue. I have the feeling that having those parenthesis make the regex more readable. Maybe we shouldn't raise an issue if the non-capturing group is part of an alternation?

Example:
```
(?:[<>]+ \w+ Intron \d+ [<>]+)|(?:\.+))
```

Reply:
This problem affects all analyzers that use the Finder. This change must be made on the analyzer-commons side. For this reason, we will create a ticket for analyzer-commons and implement it in a timely manner.